app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt

```diff
@@ -260,6 +264,223 @@ class ConversationTest {
         }
     }
 
+    @Test
+    fun testUpdateConversationAssigneeAndVerifyFromDashboard() {
+        val tempUser = getRandomKmUser()
+        val latch = CountDownLatch(1)
+        var groupId = 0
+        val botIds = listOf("inline-code-34rpc")
+        val updateBotAssigneeId = "kk-3s8r3"
+        val agentIds = listOf("prateek.singh@kommunicate.io")
```

Comment:
Don't push the email or other details like this create a Utility class and store these variables there.

Reply:
As discussed: I will be different test cases.